windows/generate_wazuh_msi.ps1

```diff
@@ -38,25 +44,12 @@ if ($PSversion -eq $null) {
     $PSversion = 1 # $PSVersionTable is new with Powershell 2.0
 }
 
-## Checking arguments
-if($READY_TO_RELEASE -eq ""){
-    "-READY_TO_RELEASE is required. Try -help to display arguments list."
-    Write-Host "Press any key to continue ..."
-    $x = $host.UI.RawUI.ReadKey("NoEcho,IncludeKeyDown")
-    Exit
-}
-
 function ComputeMsiName() {
 
     ## Checking arguments
     if($OPTIONAL_REVISION -eq ""){
         Write-Host "-OPTIONAL_REVISION empty. Using default value."
-        if($READY_TO_RELEASE -eq "yes"){
-            $OPTIONAL_REVISION="1"
-        }
-        else{
-            $OPTIONAL_REVISION = Get-Content REVISION
-        }
+        $OPTIONAL_REVISION = "1"
```

Comment:
If not revision specified use `0.0.0.default`.

Reply:
As discussed with @BraulioV , it is more comfortable to leave the default revision at value 1.